### PR TITLE
Add server status to main menu

### PR DESCRIPTION
**Pull request by veteran29**
**When merged this pull request will:**
- title

### CODE REVIEW

File: extension/src/lib.rs [hunk 1]
```diff
@@ -13,12 +13,15 @@ lazy_static! {
         .unwrap_or_else(|_| String::from("https://boderator.armaforces.com/api"));
     static ref ATTENDANCE_API: String = std::env::var("AF_ATTENDANCE_API")
         .unwrap_or_else(|_| String::from("https://armaforces.com/api"));
+    static ref SERVER_API: String = std::env::var("AF_SERVER_API")
+        .unwrap_or_else(|_| String::from("https://server.armaforces.com:8888"));
```
Comment (veteran29) [inline extension/src/lib.rs:17]: TODO we could hide this behind some sort of short living cache to not strain the game server with requests

**Comment by veteran29** [comment]
image

File: addons/extension_main_menu/functions/fnc_onLoad.sqf [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#define DEBUG_SYNCHRONOUS
```
Comment (3Mydlo3) [inline addons/extension_main_menu/functions/fnc_onLoad.sqf:1]: Is this debug needed in production? The same applies to other files.

File: addons/extension_main_menu/functions/fnc_updateServerStatus.sqf [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+    _totalW > _maxW
+} do {
+    _modlist = _modlist select [0, count _modlist - 2];
+};
+///-------------
+
+private _statusText = format [
+    ["<t size='1.1'>",
+        format ["<br/>%1:  <t size='1'>%2</t>", LLSTRING(Status), '%1'],
```
Comment (3Mydlo3) [inline addons/extension_main_menu/functions/fnc_updateServerStatus.sqf:53]:
```suggestion
        format ["<br/>%1: <t size='1'>%2</t>", LLSTRING(Status), '%1'],
```

**Review by 3Mydlo3 — APPROVED** [review]
Very nice. Such a little thing, so much tweaking. :P

File: addons/extension_main_menu/functions/fnc_onLoad.sqf (hunk 2 above)
Comment (veteran29) [reply addons/extension_main_menu/functions/fnc_onLoad.sqf:1]: It's not a debug, this changes how CBA logging works: https://github.com/CBATeam/CBA_A3/blob/9191ad571f4e31b6e7c8e17a407f0a2b2296d2c8/addons/main/script_macros_common.hpp#L173

By default the CBA logging is done via scheduled loop, I guess this is a remnant from the times when logging was done in game thread and could cause fps drops when done a lot in unscheduled. (https://github.com/CBATeam/CBA_A3/blob/1f780aee7f291e7598e0855bc8ba38b1df5b0d67/addons/diagnostic/fnc_log.sqf)

File: addons/extension_main_menu/functions/fnc_updateServerStatus.sqf (hunk 3 above)
Comment (veteran29) [reply addons/extension_main_menu/functions/fnc_updateServerStatus.sqf:53]: Double space is done intentionaly to align the text, looks better to me.

**Pull request merged**